fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/flaps_computer.rs

```diff
@@ -143,27 +146,28 @@ impl SlatFlapControlComputer {
     pub fn update(
         &mut self,
         context: &UpdateContext,
-        csu_monitor: &CSUMonitor,
         flaps_feedback: &impl PositionPickoffUnit,
         slats_feedback: &impl PositionPickoffUnit,
     ) {
-        self.flaps_conf = self.generate_configuration(csu_monitor, context);
+        self.csu_monitor.update(context);
+
+        self.flaps_conf = self.generate_configuration(&self.csu_monitor, context);
```

Comment:
`generate_configuration` can now access `csu_monitor` itself so we don't need to pass it as an argument anymore.